mne/defaults.py

```diff
@@ -83,9 +87,7 @@ def _handle_default(k, v=None):
     """Avoid dicts as default keyword arguments.
 
     Use this function instead to resolve default dict values. Example usage::
-
         scalings = _handle_default('scalings', scalings)
-
```

Comment:
As above.

Reply:
If I don't remove this my tests fail?

Reply:
It shouldn't cause tests to fail to leave these lines in (otherwise all PRs would be failing due to this). Maybe you added some whitespace to them, and that's why flake8 complained?

Reply:
ok reverted